src/transform.js

```diff
@@ -8,14 +8,15 @@ import * as parents from "./parents.js";
  * This function will not modify the root node of the input AST.
  *
  * @param {object | object[]} node AST node or array of nodes
- * @param {function(object, string, object?)} transformCallback
+ * @param {Object.<string, function> | function(object, string, object?)} transformCallback A map of node types to callbacks, or a single callback that will be called for all node types
```

Comment:
Since this is not always a callback, I'd call it something more general like "mapping" for map() and "transformation" for transform().

Reply:
Sounds good, I was looking for a better naming but couldn't decide on one